bosun.md

```diff
@@ -0,0 +1,44 @@
+2. Read through the agenda just to familiarize yourself with the content to assign timeboxes. [repeat a few times in the days leading to the meeting if it’s a live agenda]
+3. Post the draft agenda at least 3 days before with a call for final topics and a clear timebox to the kubernetes-dev mailing list. This will act as a reminder and put the agenda on the community’s radar, too.
+- Note that the dev mailing list is the source of truth. While a Slack notification is nice and helpful, the mailing list is the main communication channel.
+- Also post to the #steering-committee, #kubernetes-dev, #chairs-and-techleads slack channels
+6. Make sure there’s someone who volunteers from contribex to act as the meeting host to mute folks, verify recording, note taker and other zoom admin tasks.
+7. If no one steps up for note taking, you’re it.
+8. Do we want to poll the attendees with any questions? Do we want to take any live community voting for pulse checks? (not decisions) zoom has these features!
+9. DAY OF: Join ten minutes early to make sure all audio, video, and zoom administration (including capability to record) is right.
+10. Make sure that you have cleared your desktop and can be prepared to present if needed. Sometimes AV problems will require you to step in and present.
+
+## During
+### Starting:
+1. Before recording the meeting, tell the group “This is a Kubernetes meeting that will be posted publicly on YouTube, so please be mindful that what you say is being recorded.
+2. Please also mute if you are not speaking.” Talk about the process of waiting for n Steering members to arrive before you start.
+3. Call on others to add things to the agenda and start recording
+#### After recording starts:
+1. Hello! Welcome! [introduce yourself]
+2. Mention that we have a Code of Conduct, so be excellent to each other.
+3. Ask for a note-taker if you don't have one, ask steering members to put [steering] in their zoom display name for easy counting.
+4. Keep an eye for questions in the Zoom chat.
+5. Discuss that we may ask for further context, docs, etc. so that we can better understand larger issues or items. If there is a case where we don’t have enough or not prepared, we will set up a separate meeting, bump to the mailing list for further conversation, or bump to a next meeting.
+### Mid-meeting:
+1. Start with current steering projects in flight / stand up with steering folks
+2. Go into specific agenda topics
+3. If space, go through project board and assign owners. Do triage.
+
+## Ending:
+4. At end, notify everyone the notes will be posted soon to the mailing list. "Take some time to update the agenda or comment before I send it."
+5. Thank everyone for their time.
+
+## After:
+
+- Clean up any parts of the agenda that got mis-formatted during the meeting.
+- Set any issues that arise from the meeting
```

Comment:
We used to use milestones on a two-month cycle to try and signify what was on our radar / in flight, do we want to try that again?

Alternatively we could say if an issue has an assignee, they're responsible for keeping it in-flight.

Reply:
Why not both :)

Assignees to ensure someone is responsible for an issue, milestones for easier tracking of how things are progressing.

Reply:
I added a lot of this to the mid-meeting triage section. let me know what you think about it there. cleaned up the reference here too.